src/oemof/solph/_energy_system.py

```diff
@@ -32,12 +34,51 @@ class EnergySystem(es.EnergySystem):
     oemof.network directly.
     """
 
-    def __init__(self, **kwargs):
+    def __init__(
+        self, timeindex=None, timeincrement=None, timemode="implicit", **kwargs
```

Comment:
As "explicit"/"implicit" is a binary decision, maybe an option "infer_additional_timestamp=True" would be an alternative option. (I have the feeling that hiding the extra time step from the results is actually a bit misleading.)